pages/api/v1/auth/actions/addProfile.ts

```diff
@@ -64,9 +64,9 @@ async function handler(req: NextApiRequest, res: NextApiResponse<TokenType | Mes
 		title: title || ''
 	};
 
-	const updated_token = await authServiceInstance.getSignedToken({ ...user, profile, username });
+	const updated_token = await authServiceInstance.getSignedToken({ ...user, custom_username: custom_username, profile, username });
```

Comment:
custom_username should have the default value of false here

Reply:
fixing